src/Microsoft.AspNetCore.Mvc.Razor/RazorPageOfT.cs

```diff
@@ -38,6 +38,12 @@ public TModel Model
         public ViewDataDictionary<TModel> ViewData { get; set; }
 
         /// <summary>
+        /// Gets or sets the expression text cache for model expressions.
+        /// </summary>
+        [RazorInject]
+        public ExpressionTextCache ExpressionTextCache { get; set; }
```

Comment:
Good point @rynowak. @mnltejaswini please make this property `private`. That'll work because `RazorPageActivator` includes non-`public` properties.